This tweaks the timeout for parent queries when they spawn a child query
to solve a glueless NS or incomplete CNAME; I'm increasing it to get fewer
SERVER FAILs when resolving complex names.

--- deadwood-2.9.03/src/DwRecurse.c.orig	2010-08-03 15:19:47.000000000 -0700
+++ deadwood-2.9.03/src/DwRecurse.c	2010-08-03 15:20:09.000000000 -0700
@@ -2667,7 +2667,7 @@
         }
         make_remote_connection(new_conn_num,(unsigned char *)packet->str,
                         packet->len,rem[new_conn_num].query,INVALID_SOCKET);
-        rem[conn_number].die = get_time() + ((int64_t)timeout_seconds << 10);
+        rem[conn_number].die = get_time() + ((int64_t)timeout_seconds << 12);
         rem[conn_number].child_id = new_conn_num;
         /* Set new connection parameters */
         rem[new_conn_num].die = get_time() + ((int64_t)timeout_seconds << 8);
